assets/app/view/game/corporation.rb

```diff
@@ -174,6 +174,11 @@ def render_title(bg = nil)
           children << h(:img, logo_props)
         end
 
+        if @game.second_icon(@corporation)
```

Comment:
why is this needed?

Reply:
This was put in so that an icon can be displayed on associated minors to make it clearer which major that minor will grow up into.  It was requested in #8340, and I thought it sounded like a good idea.